migrations/20190723034059-add-type-to-expense-table.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Expenses', 'type', {
+      type: Sequelize.STRING(),
```

Comment:
Yeah I did this originally and it would be my preferred option, but I couldn't work out how to make the down migration work because it doesn't remove the enum.

I'll have another look at it. I might just have to write some raw sql.

Reply:
Enums are good if that's not too complicated. Apparently this can be done with (haven't tried):

```es6
return queryInterface.sequelize.query(
  'DROP TYPE "EnumName";'
);
```

Crazy that sequelize doesn't provide a helper for this :fearful: